mongo-setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Can we add this to the `README.md` so that people know that they need this to be run before starting?